connectors/src/resources/slack_configuration_resource.ts

```diff
@@ -185,6 +185,18 @@ export class SlackConfigurationResource extends BaseResource<SlackConfigurationM
     return new Ok(undefined);
   }
 
+  async setWhiteListedChannelPattern(pattern: string) {
```

Comment:
How do we delete pattern? Should allow null

Reply:
Indeed, was setting it to empty string instead. Correcting it.